pkg/kubelet/heartbeat.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+
+	"k8s.io/client-go/util/clock"
+)
+
+const (
+	// These constants should be reviewed and updated very carefully.
+	// nodeStatusUpdateRetry the normal default for how many times kubelet retries when posting node status failed.
```

Comment:
NodeStatusUpdateRetry is how many times the kubelet retries to post its node status after a failed attempt.